lib/FusionInventory/Agent/Task/Inventory/Generic/Rudder.pm

```diff
@@ -42,19 +44,65 @@ sub doInventory {
     # Get agent capabilities
     my @agentCapabilities = _listAgentCapabilities();
 
+    my $customProperties = _getCustomProperties(logger => $logger);
+
     my $rudder = {
         HOSTNAME => $hostname,
         UUID => $Uuid,
         AGENT => \@agents,
         SERVER_ROLES => { SERVER_ROLE => \@serverRoles },
         AGENT_CAPABILITIES => { AGENT_CAPABILITY => \@agentCapabilities },
+        CUSTOM_PROPERTIES => $customProperties,
     };
 
     $inventory->addEntry(
         section => 'RUDDER', entry => $rudder
     );
 }
 
+sub _getCustomProperties {
+    my (%params) = @_;
+    my $logger   = $params{logger};
+
+    my $custom_properties_dir = ($OSNAME eq 'MSWin32') ? 'C:\Program Files\Rudder\local\hooks.d' : '/var/rudder/local/hooks.d';
+    my $custom_properties;
+    if (-d "$custom_properties_dir") {
+        my @custom_propertes_list = ();
```

Comment:
I think you missed an `i` to say `properties` in `@custom_propertes_list`

Reply:
indeed:)